### DISCUSSION TITLE
Die selbstgebaute Media-Player-Funktion

### DISCUSSION THREAD

**Discussion by PetermitPan** [Blackymas/NSPanel_HA_Blueprint#306] [Show and tell]
Hallo zusammen,

großes Lob an die Entwickler dieses Projekts!

Ich stelle mal meine selbstgebaute Media-Player-Funktion vor.

Ich habe mir zu nutzen gemacht, dass man Szenen über die "Buttons" steuern kann. Dazu wurden im HA Szenen angelegt. In den Szenen ist immer die Entität meiner Automatisierung hinterlegt. In meinem Fall Musiksteuerung.

![grafik](https://user-images.githubusercontent.com/113134631/216967789-a789e8c1-85a4-49ae-9fb9-2219cd38bfbe.png)

![image](https://user-images.githubusercontent.com/113134631/216968068-e834a29d-c033-405c-9796-0aa77b3275e4.png)

Dann habe ich eine Automatisierung angelegt. In dieser habe ich meine vier Szenen mit eingebunden.

[Musiksteuerung (1).pdf](https://github.com/Blackymas/NSPanel_HA_Blueprint/files/10662954/Musiksteuerung.1.pdf)

Und entsprechend der Szenen gibt es auf dem Display Buttons.

![NSPanel](https://user-images.githubusercontent.com/113134631/216971144-0d959bae-bfde-47d1-b524-e8bc11b7f9e0.jpg)

**Comment by Blackymas** [comment]
thanks for sharing
